grep.c

```diff
@@ -57,144 +54,13 @@ int readnumberfromfile(const char* fname)
 
 void readGravityFiles(void)
 {
-	// Get number of domains being blocked by wildcards
-	readWildcardsList();
-	if(counters.wildcarddomains > 0)
-	{
-		logg("Wildcard blocking list entries: %i", counters.wildcarddomains);
-	}
-	else
-	{
-		logg("No wildcard blocking list present");
-	}
+	// Counting of other blocking domains is done
+	// while these lists are read
```

Comment:
Not sure what this comment means. Maybe the function needs to be renamed or inlined?